File: 프로그래머스/0/181838. 날짜 비교하기/날짜 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> date1, vector<int> date2) {
    int answer = 0;
    for(int i = 0; i < date1.size(); i++)
    {
        if(date1[i] < date2[i])
        {
            answer = 1;
            break;
        }
        else if(date1[i] == date2[i])
            answer = 0;
        else
        {
            answer = 0;
            break;
        }
    }
    return answer;
}

File: 프로그래머스/0/181838. 날짜 비교하기/날짜 비교하기.cs
using System;

public class Solution {
    public int solution(int[] date1, int[] date2) {
        int answer = 1;
        int result = 0;
        for(int i = 0; i < date1.Length; i++)
        {
            result += date2[i] - date1[i];
            if(result < 0)
            {
                answer = 0;
                break;
            }
            else if(result > 0)
            {
                answer = 1;
                break;
            }
        }
        if(result == 0)
            answer = 0;
        return answer;
    }
}